docs/Jira-Project-Migrations.md

```diff
@@ -0,0 +1,105 @@
+This guide is based on Jira Server version 7.6.1.
+
+## Pre-migration Notes
+
+*  Naming references:
+   *  `Source Jira Instance` - Jira instance containing the projects to be imported
+   *  `Target Jira Instance` - Jira instance that will be the destination of the import
+*  It is required that the source and target Jira instances are at the same version.  If the versions are different, an upgrade must be performed to bring both Jira instances to the same version.
+*  Ensure the target Jira instance contains all the necessary plugins that are required by the imported project(s).
+*  It is recommended that a test migration be performed initially on a test Jira instance that is a replica of the final target Jira instance.  This will help to ensure most issues are resolved before the final migration is performed.
+
+## Project Migration Outline
+
+*  [Create Full Backup Of Both Jira Instances](#create-full-backup-of-both-jira-instances)
+*  [Create Empty Project In Target Jira Instance](#create-empty-project-in-target-jira-instance)
+*  [Create Matching Configurations In Target Jira Instance](#create-matching-configurations-in-target-jira-instance)
+*  [Create Users/Groups/Roles In Target Jira Instance](#create-usersgroupsroles-in-target-jira-instance)
+*  [Initiate Project Import](#initiate-project-import)
+
+## Create Full Backup Of Both Jira Instances
+
+Before initiating the project migration, it is important to create a backup for both source and target Jira instances.  The source backup will be used to import the project(s) and the destination backup will serve as a fallback option if the project import causes the target Jira instance to become unusable  or unrecoverable.
+
+Backup Steps:
+*  Log into the web management console using an account with Jira administrator privileges.
+*  Click the gear wheel icon at the top right and select `System` under `JIRA ADMINISTRATION`.
+*  In the Administration page, select `Backup system` under `IMPORT AND EXPORT`.
+*  Type in a name for the backup file and click `Backup`.
+
+For a default Jira installation, the backup file should be in `/var/atlassian/application-data/jira/export`.  Repeat the above steps on the source and target Jira instances.
+
+## Create Empty Project In Target Jira Instance
+
+The Jira import mechanism requires the target Jira instance to have an existing project to serve as a receiver for the imported data.  The project must have the same project `Key` value as the source project being imported.
+
+Steps to create an empty project:
+*  Log into the web management console using an account with Jira administrator privileges.
+*  Select `Projects` on the top menu and then `Create project`.
+*  Select the project format that matches the source project and click `Next`.
+*  Select the default Issue Types and Workflow and click `Select`.
+*  Enter values for the `Name`, `Key`, select the `Project Lead`, and click `Submit`.  **IMPORTANT: The Key value must match the source project**.
```

Comment:
As with prior note: ought this be an ordered list?

Reply:
Same